CHANGES.md

```diff
@@ -28,6 +28,8 @@
 
 * Docker container unable to curl its own VIP (DCOS-45115)
 
+* Number of concurrent subscribers to Mesos master operator API is now capped to 1000 by default, with a flag to configure (DCOS_OSS-4164)
```

Comment:
I would clarify that it is a Mesos flag and not a DC/OS flag.

Do we have any plans to add a DC/OS flag to configure the maximum number of concurrent subscribers?